README.md

```diff
@@ -498,6 +498,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Ayomikun Wahab](https://github.com/Mkzay)
   - [Ayondip Jana](https://github.com/detronetdip)
   - [AyoOluwa Israel Adeleke](https://github.com/AyoOluwa-Israel)
+  - [Ayush](https://github.com/ayushsgithub)
```

Comment:
Please add your name in line 516, after Ayush Raj.
Let us know if you need any help with that. 🙂

Reply:
Ok, Done